app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::API
+  respond_to :json
```

Comment:
Try to add this also here
`include ActionController::MimeResponds`
and check of works without overriding controller